### PR TITLE
Change maintainer to Github user instead of organisation

### PR DESCRIPTION
**Pull request by FuzesiMate**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by FuzesiMate** [comment]
@conda-forge-admin, please rerender

**Comment by FuzesiMate** [comment]
@conda-forge-admin, please ping core team

Please merge this PR to give maintainer rights to my personal GitHub user.

**Comment by synapticarbors** [comment]
@FuzesiMate -- could you 
- increment the build number from `0` to `1`
- Under `requirements` - `run` and `host` change `- python` to `- python =2.7|>=3.4` to satisfy the linter and match what is in the `setup.py`.

Then I'd be happy to merge.

**Comment by FuzesiMate** [comment]
> @FuzesiMate -- could you
> 
> * increment the build number from `0` to `1`
> * Under `requirements` - `run` and `host` change `- python` to `- python =2.7|>=3.4` to satisfy the linter and match what is in the `setup.py`.
> 
> Then I'd be happy to merge.

@synapticarbors Thank you for your help! I have increased the build number and added lower bound to Python version.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,8 +11,8 @@ source:
   sha256: 25462a009dd7fa9a48520c56a691813a62919a6c842784d6ed564fe7f9f9bcf8
 
 build:
-  number: 0
-  noarch: python
+  number: 1
+  noarch: python =2.7|>=3.4
```
Comment (synapticarbors) [inline recipe/meta.yaml]: Sorry if I wasn't clear. The version restriction just goes under the `requirements` section for both `host` and `build`, not the `noarch`

File: recipe/meta.yaml [hunk 2]
```diff
@@ -21,7 +21,7 @@ requirements:
     - python
```
Comment (synapticarbors) [inline recipe/meta.yaml]: This is the `build` section I was talking about.

File: recipe/meta.yaml (hunk 1 above)
Comment (FuzesiMate) [reply recipe/meta.yaml]: Sorry, my bad, you were clear, I accidentally modified the wrong line.

**Pull request merged**